project/Dependencies.scala

```diff
@@ -187,6 +188,10 @@ object ExternalDependencies {
   val swaggerDependencies = Seq(
     "com.jakehschwartz" %% "finatra-swagger" % versions.finatra
   )
+
+  val parseDependencies = Seq(
+    "com.lihaoyi" %% "fastparse" % versions.fastparse
```

Comment:
Are we confident that this parser is the right one to use long term?

Reply:
I am not particularly attached to it, would happy if people would rather use https://github.com/sirthias/parboiled2 or something else?

Reply:
I know parboiled better, but up to you & @jamesgorrie to work out what's best.

Reply:
Have implemented this with parboiled and written some comparisons: https://github.com/wellcometrust/catalogue/pull/73